App.py

```
@@ -0,0 +1,137 @@
+import streamlit as st
+import streamlit as st
```

Ruff (0.12.2)

Redefinition of unused `st` from line 1

Remove definition: `st`

(F811)